packages/requisitions/src/RequestRequisition/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -98,7 +98,7 @@ const useStatusChangeButton = () => {
     onConfirm: onConfirmStatusChange,
   });
 
-  // When the status of the invoice changes (after an update), set the selected option to the next status.
+  // When the status changes (after an update), set the selected option to the next status.
```

Comment:
Yes, all of this cold just be one button. I've kept the component in case the status list changes again, or if finalising becomes possible ( currently it isn't, we aren't allowed to modify a 'sent' requisition )